docker/linux/Makefile

```diff
@@ -19,7 +19,7 @@ build: $(QT_ARCHIVE)
 		--build-arg="RNATIVE_VERSION=$(call __toolversion, react_native_desktop)" \
 		--build-arg="CMAKE_VERSION=$(call __toolversion, cmake)" \
 		--label="commit=$(GIT_COMMIT)" \
-		-t $(IMAGE_NAME) .; \
```

Comment:
@jakubgs this was a typo, right?

Reply:
Probably a leftover from when something happened after `docker`.